azurerm/resource_arm_metric_alertrule_test.go

```diff
@@ -34,6 +34,7 @@ func TestAccAzureRMMetricAlertRule_virtualMachineCpu(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMMetricAlertRuleExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "enabled", "false"),
+					resource.TestCheckNoResourceAttr(resourceName, "$type"),
```

Comment:
this is looking for the field `$type` rather than the Tag `$type` - as such this won't work. Instead this wants to check for `tags.$type`

Reply:
Good catch.